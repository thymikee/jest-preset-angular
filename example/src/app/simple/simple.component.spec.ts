import { async, ComponentFixture } from '@angular/core/testing';

import {ConfigureFn, configureTests} from '@lib/testing';

import { SimpleComponent } from './simple.component';

describe('SimpleComponent', () => {
  let component: SimpleComponent;
  let fixture: ComponentFixture<SimpleComponent>;

  beforeEach(
    async(() => {
      const configure: ConfigureFn = testBed => {
        testBed.configureTestingModule({
          declarations: [ SimpleComponent ]
        });
      };

      configureTests(configure).then(testBed => {
        fixture = testBed.createComponent(SimpleComponent);
        component = fixture.componentInstance;
        fixture.detectChanges();
      });
    })
  );

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('snapshots', () => {
    expect(fixture).toMatchSnapshot();
  })
});
