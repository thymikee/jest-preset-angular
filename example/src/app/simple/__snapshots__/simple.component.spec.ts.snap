// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleComponent snapshot on nativeElement should be without ng-version 1`] = `
<div
  id="root-ng-internal-isolated-2"
>
  <p>
     simple works!

  </p>
</div>
`;

exports[`SimpleComponent snapshots 1`] = `
<app-simple
  __ngContext__={[Function LRootView]}
>
  <p>
     simple works!
  
  </p>
</app-simple>
`;
